node/external/blockAPI/shardBlock.go

```diff
@@ -83,6 +87,66 @@ func (sbp *shardAPIBlockProcessor) GetBlockByRound(round uint64, options api.Blo
 	return sbp.convertShardBlockBytesToAPIBlock(headerHash, blockBytes, options)
 }
 
+// GetAlteredAccountsForBlock will return the altered accounts for the desired shard block
+func (sbp *shardAPIBlockProcessor) GetAlteredAccountsForBlock(options api.GetAlteredAccountsForBlockOptions) (*common.AlteredAccountsForBlockAPIResponse, error) {
+	headerHash, blockBytes, err := sbp.getHashAndBlockBytesFromStorer(options.GetBlockParameters)
+	if err != nil {
+		return nil, err
+	}
+
+	apiBlock, err := sbp.convertShardBlockBytesToAPIBlock(headerHash, blockBytes, api.BlockQueryOptions{WithTransactions: true, WithLogs: true})
+	if err != nil {
+		return nil, err
+	}
+
+	txPool := sbp.apiBlockToTxsPool(apiBlock)
```

Comment:
Perhaps to `outportPool` (to avoid confusion with `txsPool`)?

Reply:
done